resources/assets/js/translations/extractedMessages/resources/assets/js/components/AssessmentPlan/AssessmentPlanTable.json

```diff
@@ -27,7 +27,7 @@
   {
     "id": "assessmentPlan.summary.description",
     "description": "Description of Assessment Plan Summary Section",
-    "defaultMessage": "This is a summary of the work you've done above. You'll find\n      each assessment accompanied by a consolidated list of the essential and\n      asset skills attached to it."
+    "defaultMessage": "This is a summary of the work you've done above. You'll find\r\n      each assessment accompanied by a consolidated list of the essential and\r\n      asset skills attached to it."
```

Comment:
Same as above, if code is updated from an automated tool like translation manager, only the bits that apply to this story should be staged and the rest should go up in another PR.